src/Exception/MalformedURIException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GuzzleHttp\Psr7\Exception;
+
+use InvalidArgumentException;
+
+/**
+ * Exception thrown if a URI cannot be parsed because it's malformed.
+ */
+class MalformedURIException extends InvalidArgumentException
```

Comment:
should be named `MalformedUriException` as it's also Uri as class

Reply:
Done